index.js

```diff
@@ -249,7 +272,11 @@ function fastifyView (fastify, opts, next) {
       if (!this.res.getHeader('content-type')) {
         this.header('Content-Type', 'text/html; charset=' + charset)
       }
-      this.send(toHtml(data))
+      if (options.useHtmlMinifier && (typeof options.useHtmlMinifier.minify === 'function')) {
+        this.send(options.useHtmlMinifier.minify(toHtml(data), options.htmlMinifierOptions || {}))
```

Comment:
same here

Reply:
Same here too about removing and retesting.

Reply:
Completed